tools_iuc.yaml

```diff
@@ -1820,6 +1820,18 @@ tools:
     owner: iuc
     tool_panel_section_label: Metagenomic Analysis
 
+  - name: name2taxid
+    owner: iuc
+    tool_panel_section_label: Metagenomic Analysis
+
+  - name: ncbi_gtdb_map
```

Comment:
```suggestion
  - name: gtdb_to_taxdump
```

Reply:
@SantaMcCloud this is what you mean, correct?

You are correct in the sense that the `<tool id=""` should be `gtdb_to_taxdump` as well, but it isn't. Should we fix that ...

Reply:
Yes this is what i mean. I currently not home so i can fix it later aswell if it not urgent. Should i change the wrapper if and the `.shed` name or obly the wrapper if to `gtdb_to_taxdump`? Also if you want to fix it you can also do i when you have room for it!